webpack-test.config.js

```diff
@@ -0,0 +1,52 @@
+const webpack = require( 'webpack' );
```

Comment:
Yup its webpack config specially tailored for karma so it should be there... or better if karma is not working as it should where are you heading first? Its up to you, if you like another level of indirection lets keep it.